index.html

```diff
@@ -92,7 +95,7 @@ <h2>Meet the team</h2>
             <span>CMO</span>
           </li>
           <li>
-            <img src="https://placebear.com/150/150" alt="profile for chris caribou" />
+            <imgsrc="https://placebear.com/150/150" alt="profile for chris caribou" />
```

Comment:
I think this part got accidentally changed.  You probably want to add the space again because right now I can't see Chris's picture.
```suggestion
            <img src="https://placebear.com/150/150" alt="profile for chris caribou" />
```